common/util.go

```diff
@@ -79,21 +82,19 @@ type KuiperConf struct {
 }
 
 func init() {
+	Log = logrus.New()
+	if "" != os.Getenv(KuiperSyslogKey) {
```

Comment:
I suggest to let user specify a specified value as following (case insensitive),
`"true" == os.Getenv(KuiperSyslogKey)`